stack.yaml

```diff
@@ -1,6 +1,9 @@
-flags: {}
+flags:
+  semigroups:
+    bytestring-builder: false
 packages:
 - '.'
 extra-deps:
-- aeson-0.11.0.0
-resolver: lts-5.3
+- aeson-1.0.0.0
```

Comment:
what is the immediate benefit of using the latest version of `aeson`? how about keeping the config simple and using the latest vanilla LTS instead?